portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/RemoteApps.testcase

```diff
@@ -19,26 +19,27 @@ definition {
 		if ("${testPortalInstance}" == "true") {
 			PortalInstances.tearDownCP();
 		}
-		else {
-			RemoteApps.tearDown();
-		}
```

Comment:
Let's keep the RemoteApps.teardown here. Keep the macro named tearDown as well.

Explanation: The test is only testing if Remote App can be created. Anything else that resets the test for another testcase should be put in the teardown section. If this tearDown is not working other testcases for example CanBeDeleted because there is no Remote App, then maybe we can have the tearDown macro check that there exists a Remote App first. Best case I imagine would be a while loop to check if any Remote App exists and continue to delete the apps in case there's multiple to future proof it.

Reply:
yes, good idea, I was having doubts about this one :)